### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by lianke123321**
added global conf initialization when running vpn, moved experiments_available method to vpn/cli.py, added has_experiments_to_run back, changed logging.error to logging.exception in except block to include stack trace

### CODE REVIEW

File: centinel/vpn/cli.py [hunk 1]
```diff
@@ -213,6 +214,25 @@ def create_config_files(directory):
         configuration.write_out_config(conf_file)
 
 
+def experiments_available(config):
+    logging.info("Starting to check for experiments with %s",
+                 config['server']['server_url'])
+    try:
+        user = centinel.backend.User(config)
```
Comment (ben-jones) [inline centinel/vpn/cli.py]: You also need to pull down the results from the user. This doesn't actually pull down the experiments or check the experiments list, it just creates a user if one doesn't already exist.

**Comment by rpanah** [comment]
@ben-jones, can you review this please? I think the functions added back here were removed by you if I recall correctly.

**Comment by ben-jones** [comment]
@lianke123321, per my comment, I'll merge after you also check the list of experiments available on the server.

**Comment by lianke123321** [comment]
@ben-jones I found that checking available experiments is done in centinel.backend.sync. So I guess these lines in experiments_available should just be deleted, which I already did.

Please let me know if I misunderstood you.

**Comment by ben-jones** [comment]
@lianke123321, I'm don't understand what you mean. Centinel.backend already checked if there were experiments available, so are you just going to use that code? Also, if you are dropping experiments_available, does that mean you aren't going to use this pull request? If so, you should close the pull request.

Also, I forgot to mention this before, but it's helpful if you create a new commit for each logical item in the pull request, e.g. the exception handling update and adding back experiments_available. It makes it much easier for me to review if I browse the commits individually and, in cases like this, it makes it easy to chunk part of the pull request without having to redo the commits.

**Comment by lianke123321** [comment]
No, now the only purpose of experiments_available is to call client.has_experiments_to_run, which check if the time interval since last run is greater than the frequency in scheduler.

Making separate commits is a good idea. I'll do that in the future.

**Pull request merged**